### PR TITLE
Add overwrite.js

### PR DESCRIPTION
**Pull request by vicancy**

### CODE REVIEW

File: plugins/Microsoft.DocAsCode.Build.MemberLevelManagedReference/resources/ManagedReference.overwrite.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.
+var common = require('./ManagedReference.common.js');
+
+exports.getOptions = function (model) {
+  var ignoreChildrenBookmarks = model._splitReference && model.type && common.getCategory(model.type) == 'ns';
```
Comment (hellosnow) [inline plugins/Microsoft.DocAsCode.Build.MemberLevelManagedReference/resources/ManagedReference.overwrite.js]: >== [](start = 102, length = 2)

===?

**Comment by hellosnow** [comment]
:shipit:

**Comment by superyyrrzz** [comment]
:shipit:

File: src/docfx.website.themes/common/ManagedReference.common.js [hunk 2]
```diff
@@ -39,22 +39,24 @@ exports.transform = function (model)  {
...
+        bookmarks[item.uid] = common.getHtmlId(item.uid);
+        if (item.overload && item.overload.uid) {
+          bookmarks[item.overload.uid] = common.getHtmlId(item.overload.uid);
+        }
+      });
+    }
   }
 
+  // Reference's first level bookmark should have no anchor
+  bookmarks[model.uid] = "";
```
Comment (qinezh) [inline src/docfx.website.themes/common/ManagedReference.common.js]: how about `delete bookmarks[model.uid]`?

Comment (vicancy) [inline src/docfx.website.themes/common/ManagedReference.common.js]: It's a must have, we have a postprocessor to ValidateBookmark

**Pull request merged**